src/style.css

```diff
@@ -0,0 +1,72 @@
+html {
+  --button-hight: 19px;
+  --button-width: 19px;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+.input_line {
+  width: 92%;
+  padding-left: 13.8px;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  border: 0;
+  outline: none;
+  cursor: default;
+}
+
+.input_line_small {
+  padding-left: 6.8px;
+  font-size: 14px;
+}
+
+.button {
+  position: relative;
+  top: 4.4px;
+  padding: 0;
+  border: 0;
+  background-color: transparent;
+  height: var(--button-hight);
+  width: var(--button-width);
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+}
+
+.button-small {
+  top: 2.1px;
+  height: calc(var(--button-hight) * 0.57);
```

Comment:
why do you need vars and then use custom numbers to multiply?
set values directly or use separate vars for `button-small`